### ISSUE TITLE
[VT-132] Repeating "Negative repeat count does nothing" errors

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#153]
_Jiri Vyskocil on 2017-04-21T11:12:31Z says:_ 

 Upon startup, vit shows two errors interspersed through regualer output:

Negative repeat count does nothing at /usr/local/bin/vit line 1989.            Negative repeat count does nothing at /usr/local/bin/vit line 775.

These get repeated on almost any keypress, rendering the interface unreadable.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-04-21T11:12:31Z
Modified: 2017-05-03T12:57:49Z
```

**Comment by taskwarrior** [comment]
_Jiri Vyskocil on 2017-04-21T11:15:16Z says:_ 

 Now I see it happens when the task is too long to fit the window size (i.e. when 'task list' would say "The report has a minimum width of 102 and does not fit in the available width of 80.")

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2017-05-03T12:57:49Z says:_ 

 Thanks for this report. I believe this bug is fixed on the VIT 1.3 branch. Can you please test? You can check out the 1.3 branch after cloning from

git clone https://git.tasktools.org/EX/vit.git

**Comment by thomasrebele** [comment]
I had the same problem with version 1.2-4 (from the Debian repository)
Installing from this repository fixed the problem (vit-1.3.dev (12d03e9))
I would vote for closing this issue

**Comment by scottkosty** [comment]
@thomasrebele thank you for that confirmation! Help like that is much appreciated, especially since currently the GitHub issues for this project are in a disorganized state (because of my fault).
